wirepas_mesh_messaging/set_scratchpad_target.py

```diff
@@ -35,13 +35,16 @@ class SetScratchpadTargetAndActionRequest(Request):
                 param(int): raw parameter for action
                 delay(ProcessingDelay): delay for WITH_DELAY action
         req_id (int): unique request id
+        time_ms_epoch(int): timestamp in ms of request generation
     """
 
     def __init__(self,
                  sink_id,
                  target,
-                 req_id=None, **kwargs):
-        super(SetScratchpadTargetAndActionRequest, self).__init__(sink_id, req_id, **kwargs)
+                 req_id=None,
+                 time_ms_epoch=None,
+                 **kwargs):
+        super(SetScratchpadTargetAndActionRequest, self).__init__(sink_id, req_id, time_ms_epoch=time_ms_epoch, **kwargs)
```

Comment:
do we need to overload the constructor with time_ms_epoch? We should be able to pass in the time_ms_epoch parameter without having to add this parameter in the super constructor call.